lib/src/command/get.dart

```diff
@@ -60,6 +67,50 @@ class GetCommand extends PubCommand {
     );
   }
 
+  bool shouldRunPostGetHook() {
+    final flutterRoot = Platform.environment['FLUTTER_ROOT'];
+    if (flutterRoot == null) {
+      return false;
+    }
+    if (!fileExists(p.join(flutterRoot, 'version'))) {
+      return false;
+    }
+
+    // TODO: sky_engine check
+    final hasFlutterDependency =
+        entrypoint.root.dependencies.values.any((package) {
+      return package.name == 'flutter' &&
+          package.source.runtimeType == SdkSource;
+    });
+
+    return hasFlutterDependency;
+  }
+
+  Future<int> runPostGetHook() async {
```

Comment:
This should probably not return the exit-code but handle it on its own. Eg. by throwing an exception if non-zero.

Reply:
I've added an exception with some simple text.